src/fsharp/FSharp.Core/seq.fs

```diff
@@ -13,49 +13,9 @@ namespace Microsoft.FSharp.Collections
     open Microsoft.FSharp.Control
     open Microsoft.FSharp.Collections
 
-    module IEnumerator =
-
-
-      let noReset() = raise (new System.NotSupportedException(SR.GetString(SR.resetNotSupported)))
-      let notStarted() = raise (new System.InvalidOperationException(SR.GetString(SR.enumerationNotStarted)))
-      let alreadyFinished() = raise (new System.InvalidOperationException(SR.GetString(SR.enumerationAlreadyFinished)))
-      let check started = if not started then notStarted()
-      let dispose (r : System.IDisposable) = r.Dispose()
-
-      let cast (e : IEnumerator) : IEnumerator<'T> =
-          { new IEnumerator<'T> with
-                member x.Current = unbox<'T> e.Current
-            interface IEnumerator with
-                member x.Current = unbox<'T> e.Current :> obj
-                member x.MoveNext() = e.MoveNext()
-                member x.Reset() = noReset()
-            interface System.IDisposable with
-                member x.Dispose() =
-                    match e with
-                    | :? System.IDisposable as e -> e.Dispose()
-                    | _ -> ()   }
-
-      /// A concrete implementation of an enumerator that returns no values
-      [<Sealed>]
-      type EmptyEnumerator<'T>() =
-          let mutable started = false
-          interface IEnumerator<'T> with
-                member x.Current =
-                  check started
-                  (alreadyFinished() : 'T)
-
-          interface System.Collections.IEnumerator with
-              member x.Current =
-                  check started
-                  (alreadyFinished() : obj)
-              member x.MoveNext() =
-                  if not started then started <- true
-                  false
-              member x.Reset() = noReset()
-          interface System.IDisposable with
-                member x.Dispose() = ()
-
-      let Empty<'T> () = (new EmptyEnumerator<'T>() :> IEnumerator<'T>)
+    module Internal =
```

Comment:
I'm OK with this as a transitional step in rearranging the code with minimal diff